admin_guide/high_availability.adoc

```diff
@@ -207,40 +235,35 @@ The list of command options described here are a subset that are relevant to thi
             <number> = a number ([0-9]*).
 ////
 
-The `oadm ipfailover` command ensures that a failover pod runs on each of
-the nodes matching the constraints or label used. This pod uses VRRP (Virtual
-Router Redundancy Protocol) with link:http://www.keepalived.org/[Keepalived] to ensure that the service on the
-watched port is available, and, if needed, Keepalived will automatically float
-the VIPs if the service is not available.
-
-[[configuring-a-highly-available-routing-service]]
-=== Configuring a Highly-available Routing Service
-The following steps describe how to set up a highly-available router environment
-with IP failover:
-
-. Label the nodes for the service. This step can be optional if you run the
-service on any of the nodes in your Kubernetes cluster and use VIPs that can
-float within those nodes. This process may already exist within a complex
-cluster, in that nodes may be filtered by any constraints or requirements
-specified (e.g., nodes with SSD drives, or higher CPU, memory, or disk
-requirements, etc.).
-+
-The following example defines a label as router instances that are servicing
-traffic in the US west geography *ha-router=geo-us-west*:
-+
-====
-----
-$ oc label nodes openshift-node-{5,6,7,8,9} "ha-router=geo-us-west"
-----
-====
+[[virtual-ips]]
+=== Virtual IP Addresses
+*Keepalived* manages a set of virtual IP addresses. The admin must make sure that all of these addresses
+are accessible on the configured hosts from outside of the cluster. These addresses must not be used for any
+other purpose within the cluster.
+
+*Keepalived* on each node determines whether the needed service is running. When it is, VIPs can be supported
+and it participates in the negotiation to determine which node will serve the VIP. At a minimum
+for a node to participate, the service must be listening on the watch port.  Each VIP in the set may end up
+being served by a different node.
+
+[[kepalived-multicast]]
+=== Keepalived Multicast
+{product-title}'s *ipfailover* internally uses *keepalived*. You must ensure that multicast is enabled on the nodes
+labeled above and that the nodes can accept network traffic for 224.0.0.18 (the VRRP multicast IP address).
+When a *keepalived* daemon starts the needed `iptables` rule, if not already present, is automatically added to
+the chain specified in the --iptables-chain option.
```

Comment:
Be careful here... there are two defaults.  The one that 'oadm ipfailover' sets, which I think defaults to 'INPUT' (how do you unset it?  Send ''?)  And the one that the failover pod uses.  I believe the failover pod doesn't set a default and if there is nothing set, it does not install the rule.

I just wanted the docs to indicate there was a way to override this behavior if they did not want it.